packages/slice-machine/package.json

```diff
@@ -12,7 +12,7 @@
     "node": ">=14"
   },
   "scripts": {
-    "dev": "concurrently \"next dev\" \"npm run dev --prefix ../../e2e-projects/next\"",
```

Comment:
which project does this get run in now?

Reply:
also doesn't need to be concurrently if only one command is run

Reply:
I re-added it in another PR that will be merged in this one! Mistake sorry